### ISSUE TITLE
Ensure headings and labels are descriptive and unique

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#73]
## Description
When headings and labels are used, they must describe the topic or purpose of the element or content that they pertain to. Headings and labels must be unique unless there is sufficient context to allow users to differentiate between duplicated headings or labels. When headings and labels are not descriptive, users with cognitive impairments may have trouble finding the correct section. Users with visual impairments may have difficulty skimming the material to determine the correct section to review. Being able to predict the correct section where content will appear will save users who read slowly time and allow them to read only relevant content.

### News Grid
- [ ] "Read More" buttons should be hidden from screen readers

![43ec81f2f154e4016a39d216e404c5eb668f1148-1641299497](https://user-images.githubusercontent.com/8166060/153089564-b8e7e8f1-1561-4c65-a8ec-58bdf3d5be59.png)

**Comment by dananjohnson** [comment]
Resolved by 8866b9d
